README.md

```diff
@@ -1,37 +1,143 @@
-# skeleton
+# @fastify/throttle
 
-Template repository to create standardized Fastify plugins.
+![CI](https://github.com/fastify/fastify-throttle/workflows/CI/badge.svg)
+[![NPM version](https://img.shields.io/npm/v/@fastify/throttle.svg?style=flat)](https://www.npmjs.com/package/@fastify/throttle)
+[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](https://standardjs.com/)
 
-# Getting started
+Throttle the download speed of a request.
 
-- Click on `Use this template` above to create a new repository based on this repository.
+## Install
+```sh
+npm i @fastify/throttle
+```
+
+## Usage
+Register the plugin and, if necessary, pass custom options.
+
+This plugin will add an `onSend` hook to the Fastify instance, which will throttle the download speed of the response.
+```js
+import Fastify from 'fastify'
+
+const fastify = Fastify()
+await fastify.register(import('@fastify/throttle'), {#
+  bps: 1024 * 1024 // 1MB/s
```

Comment:
default seems to be `16384`

Reply:
this is just an example how to configure it